mocks.ts

```diff
@@ -0,0 +1,46 @@
+export interface Car {
```

Comment:
Please make interface declaration separately. Please separate if from mocks, because it will be used later on in handler